adm_program/languages/de.xml

```diff
@@ -741,6 +741,7 @@
   <string name="SYS_BACKUP_FILE">Backupdatei</string>
   <string name="SYS_BACKUP_FROM">Backup vom #VAR1# um #VAR2#</string>
   <string name="SYS_BACKUP_ONLY_MYSQL">Das Backupmodul kann nur bei einer MySQL-Datenbank genutzt werden.</string>
+  <string name="SYS_BASIC_DATA">Basic data</string>
```

Comment:
should be german

Reply:
What do you think is a good name **Basisdaten** or **Grundlegende Daten**?

Reply:
`Basisdaten`

Reply:
> @Fasse i'm not sure about `whitespace` and `empty space`. I never heard about `empty space`. And in this case the "white" from `whitespace` depends on the white paper and the missing black ink. But not sure

Ok, if you like we could stay with whitespace. Give me a short feedback.